src/App/ImportUshahidiV2/ImportMapping.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Ushahidi\App\ImportUshahidiV2;
+
+use Illuminate\Database\Eloquent\Model;
+
+class ImportMapping extends Model
```

Comment:
So I'm not sure about this... but I didn't want to add more repos/entities/etc if I didn't need to so I've just used Eloquent. On the one hand it's a chance to test using eloquent, but on the other it means we have multiple ways to do the same thing. What do you think?

Reply:
IDK , but we have to start using it at some point, so we might as well start with a small low-er impact thing like this  (compared to any other random place in platform, because it's a CLI )

Reply:
Yea. If I have time I'll play with options a bit, ie. just using Eloquent vs using Eloquent and also implementing the `Entity` interface